hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -398,3 +586,21 @@ instance Enum Transition where
 instance Bounded Transition where
   minBound = Prelude.head allTransitions
   maxBound = Prelude.last allTransitions
+
+--
+-- Prettifier
+--
+
+showRollbackInfo :: (Word, Point Block) -> String
+showRollbackInfo (rollbackDepth, rollbackPoint) =
+  toString $
```

Comment:
Should we use `Text` or `String` in `showXXX` functions? It just occurred to me because this and the function below differ.

Reply:
Seems like we are not always consistent...

Reply:
And... Sadly, it's not really our fault. Relude, cardano-api and few other libraries we use work with Text. But Quickcheck and hspec are stuck with String, so there's a bit of back and forth in our codebase.

Note that un #312, I've tried to make the status quo a bit better by making a few of such functions polymorphic in the return type, so String or Text is picked depending on the context.